mslib/mscolab/models.py

```diff
@@ -31,6 +31,21 @@
 from passlib.apps import custom_app_context as pwd_context
 from mslib.mscolab.app import db
 from mslib.mscolab.message_type import MessageType
+import sqlalchemy.types as types
```

Comment:
this shadows the builtin types

Reply:
It is not builtin but an utility function since python 3.3

It also defines names for some object types that are used by the standard Python interpreter, but not exposed as builtins like int or str are.

https://docs.python.org/3/library/types.html

fully qualified will  help us to refactor less when python occupies more names